pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -102752,7 +102752,7 @@ self: {
        description = "Wiki using happstack, git or darcs, and pandoc";
        license = "GPL";
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
+       jailbreak = true;
```

Comment:
You'll need to remove the `hydraPlatforms` line, otherwise Hydra won't build and distribute this package.

---------------------------------

Also, two other things that aren't _super_ important, but would be small improvements:

1. Instead of setting `jailbreak = true;` here in `hackage-packages.nix`, you can instead do `gitit = doJailbreak super.gitit` in the `configuration-common.nix` file like you did in https://github.com/NixOS/nixpkgs/pull/107274.
2. Please remove `gitit` from the `broken-packages` list in `configuration-hackage2nix.yaml` like in https://github.com/NixOS/nixpkgs/pull/107274.

The reason for these two changes is so that someone _could_ re-run `hackage2nix` on the release-20.09 branch, and have this `hackage-packages.nix` file regenerated exactly like it currently is.

Reply:
Thanks, that makes much more sense! Should be good now.

Reply:
@sternenseemann Thanks, that was fast!